library/experimental/mutation/pod-security-policy/allow-privilege-escalation/samples/mutation.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: Assign
+metadata:
+  name: k8spspdefaultallowprivilegeescalation
+spec:
+  applyTo:
+  - groups: [""]
+    kinds: ["Pod"]
+    versions: ["v1"]
+  location: "spec.containers[name:*].securityContext.allowPrivilegeEscalation"
+  parameters:
+    pathTests:
+    - subPath: "spec.containers[name:*].securityContext.allowPrivilegeEscalation"
+      condition: MustNotExist
```

Comment:
> MustNotExist - the path must not exist or do not mutate

What if `securityContext.allowPrivilegeEscalation: true` is set already?

Reply:
I believe in PSP if it's explicitly defined it doesn't mutate.
https://github.com/kubernetes/kubernetes/blob/master/pkg/security/podsecuritypolicy/provider.go#L185-L194